src/hooks/keyboard-navigation.js

```diff
@@ -47,33 +69,40 @@ function isElementVisible(element) {
  * [2] https://www.w3.org/TR/wai-aria-practices/#keyboard
  *
  * @param {import('preact').RefObject<HTMLElement>} containerRef
- * @param {object} options
- *   @param {boolean} [options.autofocus] - Whether to focus the first element
- *     in the set of matching elements when the component is mounted
- *   @param {boolean} [options.horizontal] - Enable navigating elements using left/right arrow keys
- *   @param {boolean} [options.vertical] - Enable navigating elements using up/down arrow keys
- *   @param {string} [options.selector] - CSS selector which specifies the
- *     elements that navigation moves between
+ * @param {KeyboardNavigationOptions} options
  */
-export function useArrowKeyNavigation(
+function useKeyboardNavigation(
   containerRef,
   {
+    active = true,
```

Comment:
ARGH! Stupid codecov differences between dev and CI. Local was showing 100% coverage. I'll fix this after getting PR feedback.